### PR TITLE
chore(oas): Remove openapi specification

### PR DESCRIPTION
**Pull request by DanielVoogsgerd**
This PR removes the Brane-OAS crate and and all its callsites.

This is/was a terrifying removal, it was more involved than I had expected. Feel free to take a look at the various callsites if I handled it properly.

### CODE REVIEW

File: overview/src/lib.rs [hunk 1]
```diff
@@ -49,4 +49,10 @@
 //!   - `specifications`: Defines the "Brane interface", i.e., network structs, non-config file layouts (mostly relating to user-facing files) and outward-facing traits and enums. Also contains legacy definitions for the old workflow representation.
 //!   - `brane-shr`: Defines common utilities and functions that aren't really covered by `brane-cfg` or `specifications`.
 //!   - `brane-oas`: Unused, but used to implement a parser for the [Open API](https://www.openapis.org/) specification language.
+//!
+//!   **Graveyard**
```
Comment (Lut99) [inline overview/src/lib.rs:53]: :heart:

**Comment by Lut99** [comment]
Terrifying xD but looks good! Want to do the #163 thingy first?

**Comment by DanielVoogsgerd** [comment]
Yes, we kinda have to anyway, as this is conflicting.

**Comment by Lut99** [comment]
Exciting...

**Pull request merged**